lib/components/table.vue

```diff
@@ -375,11 +396,18 @@
 
                 // Apply local pagination
                 if (perPage && !this.providerPaging) {
+                    // Grab the current page of data (which may be past filtered items)
                     items = items.slice((currentPage - 1) * perPage, currentPage * perPage);
                 }
 
                 // Update the value model with the filtered/sorted/paginated data set
                 this.$emit('input', items);
+
+                if (numOriginalPages !== numFilteredPages) {
```

Comment:
Hmmm that might be an idea, but I would have to make a few more data props to keep state while it is calculating it, since the only pace these values are available is in the _items computed prop.